src/store.js

```diff
@@ -48,11 +56,24 @@ export default new Vuex.Store({
     getCurrentUserId(state, userId) {
       state.currentUser.id = userId;
     },
+    getOtherUser(state, otherUser) {
```

Comment:
細かいのですが、複数形のほうが適かと思います。
```suggestion
    getOtherUsers(state, otherUser) {
```